apps/range-voting/app/components/Panels/VotePanelContent.js

```diff
@@ -78,11 +78,9 @@ class VotePanelContent extends React.Component {
       combineLatest(tokenContract.balanceOf(user), tokenContract.decimals())
         .first()
         .subscribe(([balance, decimals]) => {
-          const adjustedBalance = Math.floor(
-            parseInt(balance, 10) / Math.pow(10, decimals)
-          )
           this.setState({
-            userBalance: adjustedBalance,
+            userBalance: balance,
+            decimals: decimals
```

Comment:
Do we really want to have something like decimal, that is constant on each token contract into our hot state? I think this should come as a const from the contract better, what do you think?

Reply:
I'm just doing it how it's done in the A1 vote panel.

Reply:
I'm just doing it the way it's done in the A1 panels.